packages/orm/src/serde.ts

```diff
@@ -57,6 +57,8 @@ export interface Column {
   dbType: string;
   /** From the given `__orm.data` hash, return this columns value, i.e. for putting in `UPDATE` params. */
   dbValue(data: any, entity: Entity, tableName: string, fixups: InsertFixup[] | undefined): any;
+  /** Used by em.fork to create an __orm.row from an __orm.data. Should output what we would expect from a db query **/
+  rowValue(data: any): any;
```

Comment:
Wonder if we should rename `dbValue` and `rowValue` to `writeValue` and `readValue`?

`rowValue` is definitely better than `rowValue`, but I kinda like being more explicit about "this is used for writes" / "this is used for reads".